documentation/source/users/arkane/credits.rst

```diff
@@ -25,8 +25,8 @@ Developers: (rmg_dev@mit.edu)
 - Dr. R.J. Gillis
 - Dr. A.M. Payne
 - Asst. Prof. Y.-P. Li
```

Comment:
```suggestion
- Prof. Y.-P. Li
```

Reply:
I suggest remove the Asst from Alon above too but GitHub app won't let me comment on that line